code/Coding_Abgabe_I/INDEX.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<link rel="stylesheet" href="stylesheet.css">
+
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+
+<div id="mySidenav" class="sidenav">
+    <a href="./INDEX.html" id="HOME">HOME</a>
+    <a href="./IDEE.html" id="IDEE">IDEE</a>
+    <a href="./FRAGEBOGEN.html" id="FRAGEBOGEN">FRAGEBOGEN</a>
+    <a href="./ÜBERUNS.html" id="ÜBER">ÜBER UNS</a>
+    <! I want to center the Word Fragebogen but I just can't make it happen... Whit margin left and right it didn't work - or it didn't show in the browser?>

```

Comment:
In CSS, try:
`overflow-wrap: break-word;`
`word-wrap: break-word;`

Alternative, try to change the font-size so that it becomes a little bit smaller and increase the circle size slightly.